packages/protocol/contracts/utils/IntegerUtils.sol

```diff
@@ -1,10 +1,13 @@
 pragma solidity >=0.5.0 <0.6.0;
 
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
 library IntegerUtils {
-    
+    using SafeMath for uint256;
+
     function uintToBytes(uint256 num, uint256 size) internal pure returns (bytes5 output) {
```

Comment:
TODO: we can make an efficient casting method for this, but we can put that in another PR